build.gradle

```diff
@@ -849,18 +849,22 @@ project('test:system') {
 
     task startK8SystemTests(type: Test) {
         tasks.withType(Exec) { environment "tier2Type", System.getProperty("tier2Type", "nfs") }
+        tasks.withType(Exec) { environment "pravegaOperatorVersion",System.getProperty("pravegaOperatorVersion", "latest") }
```

Comment:
Can we move this code directly into exec task?, i.e directly into copyTestsToK8sCluster.
Perhaps with that approach we can simply this part of the code.

Reply:
Modified, please take a look.